dspace-server-webapp/src/main/java/org/dspace/app/rest/security/AuthorizationFeatureRestPermissionEvaluatorPlugin.java

```diff
@@ -0,0 +1,31 @@
+/**
+ * The contents of this file are subject to the license and copyright
+ * detailed in the LICENSE and NOTICE files at the root of the source
+ * tree and available online at
+ *
+ * http://www.dspace.org/license/
+ */
+package org.dspace.app.rest.security;
+
+import java.io.Serializable;
+
+import org.apache.commons.lang3.StringUtils;
+import org.dspace.app.rest.model.AuthorizationFeatureRest;
+import org.springframework.security.core.Authentication;
+import org.springframework.stereotype.Component;
+
+/**
+ * This class determines that any AuthorizationFeatureRest object can be viewed as it'll be a subresource of
+ * AuthorizationRest
```

Comment:
I'm not sure I understand the JavaDoc description of this plugin or the purpose of all these individualized plugins. Could you describe why these are needed, as they don't seem to actually be checking any permissions? Instead, they seem to be looking to simply see if the `targetType` is a specific string value.

Reply:
Could we update the JavaDocs here to match that same/similar description as the other `EvaluatorPlugin` classes?  I like the new descriptions added to other classes, but this one doesn't seem to be as informative.

Reply:
This class needs updated JavaDocs, as noted above.  I think it should say something like: "This class will handle calls made to AuthorizationRest endpoints. It will return true because access can be granted anytime it's linked from another resource."